core/src/test/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderIntegrationTest.kt

```diff
@@ -188,17 +195,48 @@ internal class DataBaseReaderIntegrationTest {
         }
     }
 
+    @Test
+    fun calculateHashTest() {
+        val dataSourceId = DataSourceId("test-data-source-id")
+        val queryId = QueryId("test-query-id")
+        val dataSourceProvider = BaseDataSourceProvider(
+            mapOf(dataSourceId to DataSourceConfiguration(
+                "jdbc:mysql://localhost:1234/test_data",
+                "test-username",
+                "test-password",
+                mapOf("test-property" to "test-property-value")
+            ))
+        )
+        val queryProvider = BaseQueryProvider(
+            mapOf(queryId to QueryConfiguration(
+                "test-query",
+                mapOf("test-query-parameter" to listOf("test-query-parameter-value-a", "test-query-parameter-value-b"))
+            ))
+        )
+        val dataBaseService = DataBaseServiceImpl(
+            dataSourceProvider,
+            queryProvider
+        )
+
+        assertEquals(-1879617647, dataBaseService.calculateHash(dataSourceId, queryId))
+    }
```

Comment:
This does not look like an integration test. We don't need to have a real connection to DB to calculate a hash for query and for the data source. Probably, we should move it somewhere else